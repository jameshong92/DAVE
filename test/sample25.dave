// test table binding4
int main() {
    rec r1 = new rec(age: 22, name:"Yang", GPA:3.50); 
    rec r2 = new rec(name:"James", age:23, GPA:3.88);
    rec r3 = new rec(age:24, GPA:3.80, name:"Min Woo");
    tbl test1 = tbl([r1, r2, r3]);
    print(test1);

    rec r4 = new rec(age: 22, GPA:3.66, name:"Micheal"); 
    rec r5 = new rec(name:"Jotaku", age:21, GPA:3.60);
    tbl test2 = tbl([r4, r5]);
    print(test2);
    test = append(test1, test2, false);
    print(test);
}