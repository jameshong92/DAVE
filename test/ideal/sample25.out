age	name	GPA	
22	Yang	3.50	
23	James	3.88	
24	Min Woo	3.80	

age	name	GPA	
22	Micheal	3.66	
21	Jotaku	3.60	

age	name	GPA	
22	Yang	3.50	
23	James	3.88	
24	Min Woo	3.80	
22	Micheal	3.66	
21	Jotaku	3.60	

